std_msgs/UInt16MultiArray inicial
---
std_msgs/UInt16MultiArray resultante
